subcommands/targets/offline-update.go

```diff
@@ -487,6 +488,7 @@ func doSignBundle(cmd *cobra.Command, args []string) {
 	} else {
 		subcommands.DieNotNil(err)
 	}
+	doCheckBundle(cmd, args)
```

Comment:
Do you really want to print all the information about the bundle in these cases, or just check the signatures?

Just asking about your real intent.
I'm fine either way as I think you know better what offline update users really need.

Reply:
My intention was to print info about the bundle in all cases. I think it can help a user to understand the bundle state and minimize usage errors.

Reply:
I'm fine with that. Just wanted to confirm it is _really_ the intent.